base/commandLine.cpp

```diff
@@ -1444,7 +1444,8 @@ static void calcMD5Mac(Common::Path &filePath, int32 length) {
 	if (!macResMan.open(fileName, dir)) {
 		printf("Mac resource file '%s' not found or could not be open\n", filePath.toString(nativeSeparator).c_str());
 	} else {
-		if (!macResMan.hasResFork() && !macResMan.hasDataFork()) {
+		Common::ScopedPtr<Common::SeekableReadStream> dataFork(Common::MacResManager::openFileOrDataFork(fileName, dir));
+		if (!macResMan.hasResFork() && !dataFork) {
```

Comment:
Isn't this mean that if a file will be in plain format, `dataFork` will always be not null here?

Reply:
Yes. Plain file is just a datafork. This doesn't change. getDataFork has the same behaviour